Capri.Web.Services/ServiceResult.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Capri.Web.ViewModels;
+
+namespace Capri.Web.Services
+{
+    public static class ServiceResult
+    {
+        private static readonly String _userErrorsTag = "User";
```

Comment:
You are creating a generic class like ServiceResult and now you're subordinates it to user?